tutorials/katacoda/getting-started/intro.md

```diff
@@ -0,0 +1,14 @@
+![logo](https://github.com/tektoncd/website/tree/master/tutorials/katacoda/getting-started/images/logo.png?raw=true)
+
+Tekton is Kubernetes-native and works well with widely-adopted CI/CD solutions,
```

Comment:
We should probably just use the blurb from the website for now:

> Tekton is a powerful and flexible open-source framework for creating CI/CD systems, allowing developers to build, test, and deploy across cloud providers and on-premise systems.

Once we have updated documentation covering these patterns we could revisit to add something like this instead.